tsl/test/expected/cagg_on_cagg_integer.out

```diff
@@ -11,11 +11,14 @@
 \set BUCKET_WIDTH_1ST 'INTEGER \'1\''
 \set BUCKET_WIDTH_2TH 'INTEGER \'5\''
 \set BUCKET_WIDTH_3TH 'INTEGER \'10\''
--- Run tests
+--
+-- Run common tests
+--
 \ir include/cagg_on_cagg_common.sql
 -- This file and its contents are licensed under the Timescale License.
 -- Please see the included NOTICE for copyright information and
 -- LICENSE-TIMESCALE for a copy of the license.
+\set VERBOSITY default
```

Comment:
This seems excessive. It will print the chunk names as well as other stuff and it is not particularly important for the tests below. If you want to print error details for a few particular cases, I suggest that you enable it for them alone and make sure that the chunk names are stable by, for example, creating a new hypertable.

Reply:
Fixed!